draft-packages/popover/docs/Popover.stories.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
Why the whole file?

Reply:
Because it's a stories file, and I think it's fine to do fake links (`href="#"`) for any stories.